sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -441,4 +441,8 @@ class DataFrameSuite extends QueryTest {
     checkAnswer(df.select(df("key")), testData.select('key).collect().toSeq)
   }
 
+  test("show") {
+    testData.select($"*").show()
```

Comment:
actually this is a bad idea. it prints a lot of garbage to console. maybe we can ignore the test and add a comment saying this is ignored but we want to make sure it compiles correctly. do the same for java